.github/workflows/docker.yml

```diff
@@ -0,0 +1,40 @@
+---
+on:
+  pull_request:
+  push:
+    branches: [main]
+  schedule:
+    - cron: '0 0 * * 1,2,3,4,5'
+  workflow_dispatch:
+
+name: docker
+
+jobs:
+  docker:
+    name: Docker actions
+    uses: RMI-PACTA/actions/.github/workflows/docker.yml@main
+
+  test:
+    name: Test
+    uses: ./.github/workflows/test.yml
+    needs: [docker]
+    secrets: inherit
+    strategy:
+      fail-fast: false
+      matrix:
+        config-name:
+          # - default_2022Q4
+          # - default_2023Q4
+          - full_params_2022Q4
+          - full_params_2023Q4
```

Comment:
NB: What's the difference between `default_*` and `full_params_*`?

Reply:
The `default_*`s use the `inherit` key (which should be the main mechanism), but that isn't setup in this repo yet (so they're commented out). the `full_` ones send what should be the same parameters after the param inheritance process runs, but does it all as raw JSON input.

The corresponding directory in [`workflow.pacta`](https://github.com/RMI-PACTA/workflow.pacta/tree/main/tests/config) give a pretty good idea.